content/blog/2021/06/camel-quarkus-release-2.0.0/index.md

```diff
@@ -0,0 +1,92 @@
+
+## New extensions and promotions to native
+
+New extensions:
+
+* [Avro Jackson](/camel-quarkus/latest/reference/extensions/jackson-avro.html)
+* [AWS Secrets Manager](/camel-quarkus/latest/reference/extensions/aws-secrets-manager.html) (JVM only)
+* [Azure CosmosDB](/camel-quarkus/latest/reference/extensions/azure-cosmosdb.html) (JVM only)
+* [Etcd3](/camel-quarkus/latest/reference/extensions/etcd3.html) (JVM only)
+* [Google Cloud Functions](/camel-quarkus/latest/reference/extensions/google-functions.html) (JVM only)
+* [Google Storage](/camel-quarkus/latest/reference/extensions/google-storage.html)
+* [jOOR](/camel-quarkus/latest/reference/extensions/joor.html) (JVM only)
+* [Kamelet Reify](/camel-quarkus/latest/reference/extensions/kamelet-reify.html) (JVM only)
+* [Protobuf Jackson](/camel-quarkus/latest/reference/extensions/jackson-protobuf.html)
+
+Extensions newly supported in native mode:
+
+* [DigitalOcean](/camel-quarkus/latest/reference/extensions/digitalocean.html)
+* [OpenStack](/camel-quarkus/latest/reference/extensions/openstack.html)
+* [XChange](/camel-quarkus/latest/reference/extensions/xchange.html)
+* [XQuery](/camel-quarkus/latest/reference/extensions/xchange.html)
+
+
+## Test coverage and closing functionality gaps
+
+In this release, we invested a lot of effort into reviewing existing tests and adding new test cases.
+We proceeded methodically, following the main Camel documentation.
+We took care to cover every use case mentioned on the Camel component pages.
+For example for the HTTP component, we went through the use cases mentioned on its component page](/components/latest/http-component.htm),
+and we checked whether they are covered by existing tests.
+All the missing scenarios were listed in a ["Test expansion" issue](https://github.com/apache/camel-quarkus/issues/2794)
+and fixed by adding the [respective tests](https://github.com/apache/camel-quarkus/commit/5c969cac27abd1af122b895fc0a7e7f26b69df25).
+
+In this way, we not only found and fixed bugs, but we also uncovered fully missing features.
+This was the case e.g. with `@org.apache.camel.EndpointInject`, `@org.apache.camel.Produce` and `@org.apache.camel.Consume` annotations from `camel-core`.
+These are supported by Camel Quarkus now - see [here](/camel-quarkus/latest/user-guide/cdi.html#_endpointinject_and_produce) and [here](/camel-quarkus/latest/user-guide/cdi.html#_consume).
+
+## Deprecations
+
+* Avro extension: `@BuildTimeAvroDataFormat` is deprecated - see the [Avro extension](/camel-quarkus/latest/reference/extensions/avro.html#_additional_camel_quarkus_configuration) page.
+
+
+## Breaking changes and migration steps
+
+Please refer to our [2.0.0 Migration guide](/camel-quarkus/latest/migration-guide/2.0.0.html).
+
+## Release notes
+
+Fixed issues:
+
+* [Milestone 2.0.0-M1](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
+* [Milestone 2.0.0-M2](https://github.com/apache/camel-quarkus/milestone/13?closed=1)
```

Comment:
miestone/16  for M2 ?

Reply:
Yep, thanks, another copy and paste mistake